app/v5/createForeignPropertyPeriodCumulativeSummary/def1/model/request/Expenses.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v5.createForeignPropertyPeriodCumulativeSummary.def1.model.request
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Json, Reads, Writes}
+
+case class Expenses(premisesRunningCosts: Option[BigDecimal],
+                    repairsAndMaintenance: Option[BigDecimal],
+                    financialCosts: Option[BigDecimal],
+                    professionalFees: Option[BigDecimal],
+                    costOfServices: Option[BigDecimal],
+                    travelCosts: Option[BigDecimal],
+                    residentialFinancialCost: Option[BigDecimal],
+                    broughtFwdResidentialFinancialCost: Option[BigDecimal],
+                    other: Option[BigDecimal],
+                    consolidatedExpenses: Option[BigDecimal])
+
+object Expenses {
+  implicit val reads: Reads[Expenses] = Json.reads[Expenses]
+
+  implicit val writes: Writes[Expenses] = (
+    (JsPath \ "premisesRunningCosts").writeNullable[BigDecimal] and
```

Comment:
Same here for the field names for premisesRunningCosts and repairsAndMaintenance.